tt_metal/common/assert.hpp

```diff
@@ -96,26 +96,27 @@ inline std::string backtrace_to_string(int size = 64, int skip = 2, const std::s
     return ss.str();
 }
 
+// fix from https://github.com/fmtlib/fmt/issues/2438
```

Comment:
This is a wrong fix. Here you know the `fmt` string at compile time, so you don't need to push it to runtime.

Reply:
fixed, please review

Reply:
Sry, I don't believe the fold function will work as intended for 2/3 overloads. Leaving with `fmt::runtime` for now